zulipterminal/cli/run.py

```diff
@@ -200,7 +200,8 @@ def main(options: Optional[List[str]]=None) -> None:
         else:
             theme_to_use = zterm['theme']
         available_themes = all_themes()
-        if theme_to_use[0] not in available_themes:
+        theme = theme_to_use[0]
```

Comment:
Based on other renaming here, `theme_name`?

This first commit feels like a combination of 'preparing for other commits' elements, the most obvious of which is working with just the theme names in run.py, and only accessing the theme itself in core.py. The other element is setting the zuliprc_path in core.py, which perhaps should go with another commit. Then you can describe the intent of this first commit in terms of just that theme_name vs theme change.